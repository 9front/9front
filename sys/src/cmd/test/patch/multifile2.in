77777
77778
77779
77780
77781
77782
77783
77784
77785
77786
77787
77788
77789
77790
77791
77792
77793
77794
77795
77796
77797
