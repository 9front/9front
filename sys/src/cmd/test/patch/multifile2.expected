77778
77779
77780
77781
77782
77789
77788
77787
77786
77785
77784
77783
77790
77791
77792
77793
77794
77796
77797
